claim_batch/services.py

```diff
@@ -342,47 +342,26 @@ def do_process_batch(audit_user_id, location_id, end_date):
                 .filter(process_stamp__lte=end_date)\
                 .filter((Q(items__product=product) | Q(services__product=product)))
-            work_data['claims'] = claims
             # adds the filter to includ only the claims according the start/stop and cieling definition 
             claims = add_hospital_claim_date_filter(claims, relative_price_g, start_date, start_date_ip, end_date, product.ceiling_interpretation)
-            # prefectch Item and services
+            work_data['claims'] = claims
 
-            claim_batch_valuation(work_data)
-
-            # 2.1 [for the futur if there is any need ]filter a calculation valid for batchRun with context BatchPrepare (got via 0.2): like allocated contribution
-            #if work_data.paymentplans:
-            #    for paymentplan in work_data.paymentplans:
-            #        if paymentplan.calculation.active_for_object(batch_run, 'BatchPrepare'):
-            #            # work_data gets updated with relevant data such as work_data.allocated_contribution
-            #            paymentplan.calculation.calculate(work_data, 'BatchPrepare'))
-            #        # 3 Generate Batchvaluation:
-            #        # 3.1 filter a calculation valid for batchRun with context BatchValuation (got via 0.2)
-            #        if paymentplan.calculation.active_for_object(batch_run, 'BatchValuate'):
-            #        # 3.2 Execute the calculation per Item or service (not claims)
-            #            paymentplan.calculation.calculate(work_data, 'BatchValuate'))
-            #else:
-            #    logger.info("no payment plan product found for product  %s for %s/%s", product_id, period, year)
-            # 4 update the claim Total amounts if all Item and services got "valuated"
-
-
-            # could be duplicates - distinct
-            claims = claims.prefetch_related(Prefetch('items', queryset=ClaimItem.objects.filter(legacy_id__isnull=True)))\
-                .prefetch_related(Prefetch('services', queryset=ClaimService.objects.filter(legacy_id__isnull=True)))
-            claims = list(claims.values_list('id', flat=True).distinct())
-            claims = Claim.objects.filter(id__in=claims)
-
-            for claim in claims:
-                remunerated_amount = 0
-                for service in claim.services.all():
-                    remunerated_amount = service.remunerated_amount + remunerated_amount if service.remunerated_amount else remunerated_amount
-                for item in claim.items.all():
-                    remunerated_amount = item.remunerated_amount + remunerated_amount if item.remunerated_amount else remunerated_amount
-                if remunerated_amount > 0:
-                    claim.valuated = remunerated_amount
-                    claim.save()
-                claim.status = Claim.STATUS_VALUATED
-                claim.batch_run = work_data["created_run"]
-                claim.save()
```

Comment:
I am ok to remove that from the batchrun BUT it need to be moved to the paymentplan (will be generic enough for all the claim batch calcrule)